packages/tools/client-debugger/client-debugger-view/src/components/DataObjectsView.tsx

```diff
@@ -18,10 +32,60 @@ export type DataObjectsViewProps = HasContainerId;
  * Dispatches data object rendering based on those provided view {@link DataObjectsViewProps.renderOptions}.
  */
 export function DataObjectsView(props: DataObjectsViewProps): React.ReactElement {
+	const { containerId } = props;
+
+	const messageRelay = useMessageRelay();
+
+	const [rootDataHandles, setRootDataHandles] = React.useState<
+		Record<string, RootHandleNode> | undefined
+	>();
+
+	React.useEffect(() => {
+		const inboundMessageHandlers: InboundHandlers = {
+			["ROOT_DATA_VISUALIZATIONS"]: (untypedMessage) => {
+				const message: RootDataVisualizationsMessage =
+					untypedMessage as RootDataVisualizationsMessage;
+
+				if (message.data.containerId === containerId) {
+					setRootDataHandles(message.data.visualizations);
+
+					return true;
+				} else {
+					return false;
+				}
+			},
+		};
+
+		function messageHandler(message: Partial<IDebuggerMessage>): void {
+			handleIncomingMessage(message, inboundMessageHandlers, {
+				context: loggingContext,
+			});
+		}
+
+		messageRelay.on("message", messageHandler);
+
+		// POST Request for DDS data in container.
+		messageRelay.postMessage({
+			type: "GET_ROOT_DATA_VISUALIZATIONS",
```

Comment:
If you pull the most recent main, you can use:
```suggestion
			type: GetRootDataVisualizationsMessageType,
```